JavaAssignment/src/ankitaG/Program9.java

```diff
@@ -0,0 +1,27 @@
+package ankitaG;
+
+public class Program9 {
+	static int num1, num2;
+
+	static void setdata(int x, int y) 
```

Comment:
Here no need of setdata method because you define this variable as instance as well static, you can directly pass a parameter to **swap(firstNum, secondNum)** 
while the declaration of this method **swap(int num1, int num2) , so we don't need to create num1 and num2 variable as instance and static.**